### PR TITLE
iTechOps - Update job-openings-data.json

### PR DESCRIPTION
**Pull request by dhawaldenny**
iTechOps - Update job-openings-data.json

### CODE REVIEW

File: public/data/job-openings-data.json [hunk 1]
```diff
@@ -231,6 +231,54 @@
...
       ]
+    },
+    {
+      "id": 7,
+      "companyName": "iTechOps Cloud Pvt Ltd",
+      "location": "Ahmedabad",
+      "bio": "iTechOps helps customers to turn creative idea's into digital softwares. iTechOps provides satisfactory, high-quality, cost-effective DevOps and Full Stack Development Consulting services. Join our growing team to learn cutting-edge tools & technologies.",
+      "companyLogoUrl": "https://itechops.com/assets/ilogo_128.jpg",
+      "companyWebsite": "https://itechops.com/",
+      "companyCareerPage": "https://itechops.com/career",
```
Comment (NishargShah) [inline public/data/job-openings-data.json:242]: Page is giving 500, please check

File: public/data/job-openings-data.json [hunk 2]
```diff
@@ -231,6 +231,54 @@
           "experience": "-"
         }
       ]
+    },
+    {
+      "id": 7,
+      "companyName": "iTechOps Cloud Pvt Ltd",
+      "location": "Ahmedabad",
+      "bio": "iTechOps helps customers to turn creative idea's into digital softwares. iTechOps provides satisfactory, high-quality, cost-effective DevOps and Full Stack Development Consulting services. Join our growing team to learn cutting-edge tools & technologies.",
+      "companyLogoUrl": "https://itechops.com/assets/ilogo_128.jpg",
```
Comment (NishargShah) [inline public/data/job-openings-data.json:240]: It is giving 404 page not found, please check

File: public/data/job-openings-data.json (hunk 1 above)
Comment (dhawaldenny) [reply public/data/job-openings-data.json:242]: @NishargShah, sorry for the inconvenience. I have fixed the issue. Please check and approve.

File: public/data/job-openings-data.json (hunk 2 above)
Comment (dhawaldenny) [reply public/data/job-openings-data.json:240]: @NishargShah, sorry for the inconvenience. I have fixed the issue. Please check and approve.

**Comment by JaldeepAsodariya** [comment]
@dhawaldenny  Published on website
- https://devfest.gdgahmedabad.com/job-openings/7

**Pull request merged**